src/EA36E0.c

```diff
@@ -1,18 +1,57 @@
 #include "dead.h"
 #include "common.h"
 
-INCLUDE_ASM(s32, "EA36E0", func_80242DE0_EA36E0);
+#define NAMESPACE EA36E0
+
+// Copy of kzn_17?
+
+#define UNK_FUNC_50_VAR1 -583.0
+#define UNK_FUNC_50_VAR2 165.0
+
+ApiStatus func_80242DE0_EA36E0(Evt* script, s32 isInitialCall) {
```

Comment:
You can totally name this, lol

Reply:
Oh yeah, I honestly just never think about naming functions - Will do.